playbooks/common/openshift-checks/health.yml

```diff
@@ -11,4 +11,4 @@
     - action: openshift_health_check
       args:
         checks:
-          - '@preflight'
+          - '@health'
```

Comment:
Yes, exactly. It makes no sense to run most preflight checks on an ongoing basis, or to run most health checks at preflight, so just separating them out.